x-pack/plugins/reporting/common/constants.ts

```diff
@@ -7,8 +7,6 @@
 
 export const PLUGIN_ID = 'reporting';
 
-export const BROWSER_TYPE = 'chromium';
```

Comment:
This const was only used in 1 place, so I took an opportunity to clean up this file by removing the const